tests/conftest.py

```diff
@@ -24,7 +28,7 @@ def chrome(debug=False):
     return driver
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="function")
```

Comment:
```suggestion
@pytest.fixture()
```
тоді скоуп вказувати взагалі може не треба, бо "функція" - це скоуп за замовчанням